static/templates/message_reactions.handlebars

```diff
@@ -1,7 +1,7 @@
 {{#each this/msg/message_reactions}}
 {{partial "message_reaction"}}
 {{/each}}
-<div class="reaction_button" title="{{t 'Add reaction...' }}">
+<div class="reaction_button" title="{{t 'Add reaction' }} (:)">
```

Comment:
There's another one here:
`static/js/emoji_picker.js:124:        elt.prop('title', 'Add reaction...');`
(You can get to it by adding two reactions and deleting the second one, or something).